### PR TITLE
updated robotjs to compile with nan-2.0.9

### PR DESCRIPTION
**Pull request by abhishekgarg120**

### CODE REVIEW

**Comment by octalmage** [comment]
Thanks for this! I have a few comments, I'll comment on the relevant lines.

File: src/robotjs.cc [hunk 1]
```diff
@@ -28,65 +28,59 @@ int keyboardDelay = 10;
...
+	size_t x = info[0]->Int32Value();
+	size_t y = info[1]->Int32Value();
 
 	MMPoint point;
 	point = MMPointMake(x, y);
 	moveMouse(point);
 	microsleep(mouseDelay);
 
-	NanReturnValue(NanNew("1"));
+//	info.GetReturnValue().Set(Nan::New(1));
```
Comment (octalmage) [inline src/robotjs.cc]: Is this commented out for a reason? This is what caused the tests to fail.

Comment (abhishekgarg120) [inline src/robotjs.cc]: This is commented out by mistake. Let me fix this

**Comment by abhishekgarg120** [comment]
This now works with the latest Nodejs v4.0.0

**Comment by octalmage** [comment]
Thanks! The io.js v3 test is still failing, but I think that's a separate issue (travis config). I'll look into it later today.

**Comment by octalmage** [comment]
Hey @abhishekgarg120! 

Could you change your .travis.yml to this and commit it to this PR? 

```
sudo: false
language: node_js
env:
  - CXX=g++-4.8
node_js:
  - 0.10
  - 0.12
  - 4.0.0
  - iojs-v2
  - iojs-v3

matrix:
  allow_failures:
    - node_js: iojs-v3
    - node_js: 4.0.0

addons:
  apt:
    sources:
      - ubuntu-toolchain-r-test
    packages:
      - libx11-dev
      - zlib1g-dev
      - libpng12-dev
      - libxtst-dev
      - g++-4.8

before_script:
  - export DISPLAY=:99.0
  - sh -e /etc/init.d/xvfb start
```

Or if you're busy, can you add me as a collaborator to your fork? 

Thanks!

**Comment by abhishekgarg120** [comment]
Sure, i'll do it, give me a minute.

**Comment by ChALkeR** [comment]
Adding to the list: https://github.com/nodejs/node/issues/2798.

**Pull request merged**